README.md

```diff
@@ -17,6 +17,19 @@ This repository maintains the templates used in conjunction with the [VS Code To
 | ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/aspnetblazor.png?raw=true)   | ASP.NET 6 Blazor Application    | ASP.NET 6.0 | C#        | ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/arm32.png?raw=true) ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/arm64.png?raw=true)  ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/x8664.png?raw=true)                                                                                                       | [aspnetBlazor](./aspnetBlazor)     |
 | ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/python3console.png?raw=true) | Python 3 Console Application    | Python 3    | Python 3  | ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/arm32.png?raw=true) ![](https://github.com/toradex/vscode-torizon-templates/blob/bookworm/assets/img/arm64.png?raw=true)                                                                                                                                                                                                             | [python3Console](./python3Console) |
 
+
+
+# Partner Templates
+
+Partner templates are contributions come from [Toradex partner network](https://www.toradex.com/support/partner-network) associates:

```

Comment:
Partner templates are contributions that come from

Reply:
done